### DISCUSSION TITLE
Matches documentation

### DISCUSSION THREAD

**Discussion by jmarbutt** [magicbell/community#156] [Questions] [answered]
Is there documentation on how to use the `matches` filter? I am looking at adding attributes for a tenant id on my users, so I can send a broadcast to all users who are a part of a tenant. Is this the best way to do it.

**Comment by unamashana** [comment, marked as answer]
Hey @jmarbutt we have deprecated matches. I'd enourage you to create topics - one per tenant, and subscribe members to it. Then. you can use the `topic: {subscribers: true}` as a recipient, and also a `topic` key to send to the subscribers. https://www.magicbell.com/docs/topics-subscriptions/overview
